src/main/java/org/zalando/nakadi/domain/Storage.java

```diff
@@ -70,6 +75,7 @@ public String toString() {
     }
 
     private String id;
+    @JsonProperty("storage_type")
```

Comment:
I don't know. Creating a second `Storage` to preserve layering but actually having very similar objects (all attributes with the same type) sometimes is might not help so much. I prefer to make a 2k lines refactoring every 6 months than being strict on layering and end up with a lot of equal classes.